option-contract-doc/scribblings/option-contract.scrbl

```diff
@@ -27,8 +27,8 @@ Returns a contract that recognizes vectors or hashes or instances of
 struct @racket[struct-id]. The data structure must match @racket[c] and pass the
 @racket[tester].
 
-When an @racket[option/c] contract is attached to a value, the value is checked against the
-@racket[tester], if @racket[tester] is a predicate. After that,
+When an @racket[option/c] contract is attached to a value, the value is wrapped in the contract
+@racket[c] and checked against the @racket[tester], if @racket[tester] is a predicate. After that,
```

Comment:
How about "when an option/c contract is attached to a value, first the contract c is attached to the value and then the result is checked against `tester`", since only some contracts do wrapping?

Reply:
That makes more sense, I was too focused on my particular use case.